sdks/python/apache_beam/runners/common.py

```diff
@@ -264,7 +264,9 @@ def __init__(self, output_processor, signature):
 
   def invoke_process(self, windowed_value, restriction_tracker=None,
                      output_processor=None):
-    output_processor = output_processor or self.output_processor
+    if not output_processor:
+      output_processor = self.output_processor
+    assert output_processor
```

Comment:
Do we need these asserts on every element?

Reply:
Removed asserts.